src/Spaces/finitedifference.jl

```diff
@@ -153,6 +153,16 @@ local_geometry_data(space::CenterFiniteDifferenceSpace) =
 local_geometry_data(space::FaceFiniteDifferenceSpace) =
     space.face_local_geometry
 
+
+z_component(::Type{<:Geometry.LatLongZPoint}) = 9
+z_component(::Type{<:Geometry.ZPoint}) = 1
+
+function get_dz(space::AbstractSpace)
```

Comment:
analogous to `coordinates_data`, `local_geometry_data`
```suggestion
function dz_data(space::AbstractSpace)
```